ksqldb-parser/src/main/java/io/confluent/ksql/parser/tree/TableElement.java

```diff
@@ -128,4 +133,13 @@ public String toString() {
         + ", namespace=" + namespace
         + '}';
   }
+
+  public static Namespace getNamespaceFromContext(final SqlBaseParser.TableElementContext context) {
+    if (context.headerType() != null) {
```

Comment:
Would it be simpler to use a `switch` statement?

Reply:
How these `...Context` objects work is for each possible token that could be in the element, there is a field that either contains an object or is `null`. So we have to go through each field to see if it's there.